package/kernel/mac80211/files/lib/netifd/wireless/mac80211.sh

```diff
@@ -998,10 +998,10 @@ get_freq() {
 	iw "$phy" info | awk -v band="$band" -v channel="[$channel]" '
 
 $1 ~ /Band/ {
-	band_match = band == $2
+	band_match = band = $2
 }
 
-band_match && $3 == "MHz" && $4 == channel {
+band_match && $3 = "MHz" && $4 = channel {
```

Comment:
This is embedded Awk code, is this change really intended?

Reply:
Fixed.